content/soda-gettingstarted/installation-using-ansible.md

```diff
@@ -57,6 +57,19 @@ A SODA release conists of various projects which have their own release cycles a
 
 Set the environment variable `HOST_IP` by using the steps below. 
 
+
+To set the HOST_IP environment variable, you first need to find your machine's IP address.
+Use the following command to display network interface information:
+
+```bash
+ifconfig
```

Comment:
ifconfig is not available on Ubuntu 18.04 and above, by default. Alternatively `ip -4 addr` will give the available network connections.